companies/morganstanley.json

```diff
@@ -0,0 +1,56 @@
+{
+    "slug": "morganstanley",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Morgan Stanley & Co International plc",
+    "address": "Legal & Compliance Division\nMorgan Stanley & Co International plc\n20 Bank Street, Canary Wharf\nLondon E14 4AD\nUnited Kingdom",
+    "email": "dataprotectionoffice@morganstanley.com",
+    "web": "https://www.morganstanley.com",
+    "sources": [
+        "https://www.morganstanley.com/disclaimers/emea-privacy-policy"
+    ],
+    "required-elements": [
```

Comment:
Are you sure that all these elements are necessary? We generally try to keep these to a minimal set so as not to expose unnecessary personal data to the company when sending a request.

I have no experience with Morgan Stanley but usually the combination of name and email or name and address are sufficient.

Reply:
You're right. The US version using CCPA requested all these details, but I've simplified it and let the request any additional data